manifests/p/Prusa3D/PrusaSlicer/2.4/Prusa3D.PrusaSlicer.installer.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Prusa3D.PrusaSlicer
+PackageVersion: 2.4
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: exe
+  Scope: machine
+  InstallerUrl: https://cdn.prusa3d.com/downloads/drivers/PrusaSlicer_Win_standalone_2.4.0.exe
+  InstallerSha256: C7D1DB2BA3754F9A64DD2E10BEF1F0B6AD28053DE5A1FFF2B0B248D7D505973B
+  InstallerSwitches:
+    Silent: /silent

```

Someone have been drunk with this switch
```suggestion
    Silent: /quiet
```